howsthewaterfront/src/components/App.js

```diff
@@ -59,12 +91,15 @@ class App extends React.Component {
     );
     return (
       <>
-        <Route exact path="/" component={LandingForm} />
-        <Route exact path="/login" component={LoginForm} />
-        <Route exact path="/signup" component={SignUpForm} />
-        <Route exact path="/searchresult" component={SearchResultForm} />
-        <Route exact path="/settings" component={Settings} />
-        <Routes childProps={childProps} />
+          <Route exact path="/" component={LandingForm} />
+          <Route exact path="/login" component={LoginForm} />
+          <Route exact path="/signup" component={SignUpForm} />
+          <Route exact path="/searchresult" component={SearchResultForm} />
+          <Route exact path="/settings" component={Settings} />
+          <Routes childProps={childProps} />
+          {/* Test to make sure apollo is fetching data as expected, will change in future to 
+          display the data appropriately */}
+          <GetBeaches />
```

Comment:
Daly, Bravo! looks good. I was able to see all the beaches. It will be a good idea to comment out the <GetBeaches /> component. You can use a console.log to print it to the console than print it to the screen.